api-specifications/properties.json

```diff
@@ -229,70 +229,229 @@
+          "optional": true,
+          "type": "#/definitions/CellStyling"
+        },
+        "dimensionValues": {
+          "description": "Styling for dimension value cells",
+          "optional": true,
+          "type": "#/definitions/CellStyling"
+        },
+        "measureValues": {
+          "description": "Styling for measure value cells",
+          "optional": true,
+          "type": "#/definitions/CellStyling"
+        },
+        "measureLabels": {
+          "description": "Styling for measure label cells",
+          "optional": true,
+          "type": "#/definitions/PartialCellStyling"
+        },
+        "totalValues": {
+          "description": "Styling for total value cells",
+          "optional": true,
+          "type": "#/definitions/PartialCellStyling"
+        },
+        "nullValues": {
+          "description": "Styling for null values cells",
           "optional": true,
-          "type": "#/definitions/HeaderStyling"
+          "type": "#/definitions/PartialCellStyling"
+        },
+        "grid": {
+          "description": "General grid styling",
+          "optional": true,
+          "type": "#/definitions/GridStyling"
         }
       }
     },
-    "ContentStyling": {
-      "description": "Holds properties for font size, font color and hover styling.",
+    "CellStyling": {
+      "description": "Properties for styling a cell",
       "kind": "object",
       "entries": {
         "fontSize": {
-          "description": "Defaults to `14`",
+          "description": "Font size in pixel value",
           "optional": true,
-          "type": "number"
+          "type": "string"
         },
-        "fontColor": {
-          "description": "Defaults to `#404040`",
+        "fontFamily": {
+          "description": "Font family",
           "optional": true,
-          "type": "#/definitions/PaletteColor"
+          "type": "string"
         },
-        "hoverEffect": {
-          "description": "Toggles hover effect",
+        "fontStyle": {
+          "description": "Font style",
           "optional": true,
-          "type": "boolean"
+          "kind": "array",
+          "items": {
+            "type": "#/definitions/FontStyleValues"
+          }
         },
-        "hoverColor": {
-          "description": "Background hover color. Uses `#f4f4f4` if no hover colors are set, is transparent if only `hoverFontColor` is set",
+        "fontColor": {
+          "description": "Font color palette",
           "optional": true,
           "type": "#/definitions/PaletteColor"
         },
-        "hoverFontColor": {
-          "description": "When only `hoverColor` is set, this is adjusted to either `#f4f4f4` or `#ffffff` for optimal contrast",
+        "background": {
+          "description": "Cell background color palette",
           "optional": true,
           "type": "#/definitions/PaletteColor"
+        }
+      }
+    },
+    "PartialCellStyling": {
```

Comment:
No idea what to call this. Maybe there is a better way to pick some props from another type?

Reply:
Yeah you could reuse CellStyling and omit the props not needed. But you still would have to name it right? I think the current name is fine.

Reply:
Yes I don't think there is a way around that